gateleen-hook/src/main/java/org/swisspush/gateleen/hook/reducedpropagation/ReducedPropagationManager.java

```diff
@@ -105,7 +105,7 @@ public void startExpiredQueueProcessing(long intervalMs) {
 
         long expireTS = System.currentTimeMillis() + propagationIntervalMs;
 
-        log.info("Going to perform a lockedEnqueue for (original) queue '" + queue + "' and eventually starting a new timer");
+        log.debug("Going to perform a lockedEnqueue for (original) queue '" + queue + "' and eventually starting a new timer");
```

Comment:
(*minor*) Should we replace all those string concatenations with placeholders? (multiple occurrences)

See also [SO question logger-slf4j-advantages-of-formatting-with-instead-of-string-concatenation](https://stackoverflow.com/questions/10555409/logger-slf4j-advantages-of-formatting-with-instead-of-string-concatenation)